### ISSUE TITLE
Automate use of Android expansion packs

### ISSUE THREAD

**Issue by JohnWeidner** [phonegap/build#433]
To me the biggest limitation of PhoneGap Build is the file size limitation.   But if PGB had a solution for using Android Expansion packs, this wouldn't be an issue as you could just put all your large assets (video, graphics, sound) in an expansion pack.    Without this feature, I've had to use the CLI tools.   But trying to get the project to build using an expansion pack has been a major pain.   To me PGB's value is all about reducing the pain and complexity of using the CLI tools.   This one feature alone would be all I need to get turned into a paying customer.

**Comment by amirudin** [comment]
+10

**Comment by faizilyas** [comment]
I agree!!! I'm already a paying customer... This is a feature which is desperately needed!!!

**Comment by amirudin** [comment]
Original Topic
http://gsfn.us/t/4q5oo

**Comment by amirudin** [comment]
Related request: http://gsfn.us/t/4qzgj

**Comment by mzBILALI** [comment]
Agree with all above...this should be standard!

**Comment by bholmes451** [comment]
I am going to vote this up as well.  I am trying to save my users as much cell data transfer as possible and make my app as responsive as possible so my app includes about 150mb of images...

**Comment by ismailhameed** [comment]
Well i know what im doing next month...canceling my membership to phone gap..what am i even paying for? A whole load of nothing?!

**Comment by zishanali123** [comment]
Too right, GET IT SORTED.

**Comment by HasanHanif** [comment]
Why would PhoneGap leave out such an important feature? It makes no sense at all. PhoneGap desperately needs to sort this ASAP.

**Comment by wildabeast** [comment]
Getting the impression some of you may want Android Expansion Pack support on build! We haven't heard a ton of noise about this feature until recently. We'll do some recon and get back asap. Your patience is appreciated as we're stretched a bit thin on PGB atm.

**Comment by Amarshotta** [comment]
PhoneGap, this is desperately needed

**Comment by faizilyas** [comment]
I needed these feature a while back and just assumed that the xapkreader plugin was what was standard and just couldn't get my head round making it work. Been battling it for ages!!! Each time i tried contacting Phonegap build I would be met with a lack of a response. So glad I'm not the only one... Phew! Good to hear you're getting onto this... but cum'on Phonegap you need to pull your socks up and sort this one out. Going to save loads of us from a headache of making that awful plugin work.

**Comment by jessemonroy650** [comment]
My impression is that this is important, BUT i also getting the impression that some of the comments are sock puppets.

**Comment by mzBILALI** [comment]
Glad you guys are sorting it. Can't wait to make my app bigger and better!

**Comment by bholmes451** [comment]
Sock puppets?  I never expected something I brought up would be so exciting to the community...  So far I've spent 6 hours trying to get the xapkreader going but still don't have any of my images coming up in my app.  I fully expect to get it figured out but have no idea how long it's going to take me and I never expected to have this type of problem with the google play store in the first place.  Please involve me when you get to working out requirements because it is very important to me to be able to implement all of this in a single phonegap/angularjs codebase.

**Comment by hasanak2** [comment]
Sock puppets... you must be joking! This is something that is needed and because you can't be bothered to do anything about it you say sock puppets! I back the need for this feature 100%... Something which is needed and would help new developers create fantastic apps. PhoneGap Build get behind this one.

**Comment by macdonst** [comment]
While you are waiting for this feature to be implemented you may want to look at the new content sync plugin which makes it easier to download additional assets.

https://github.com/phonegap/phonegap-plugin-contentsync
